templates/CRM/common/footer.tpl

```diff
@@ -31,7 +31,7 @@
 
   <div class="crm-footer" id="civicrm-footer">
     {crmVersion assign=version}
-    {ts 1=$version}Powered by CiviCRM %1.{/ts}
+    {ts}Powered by CiviCRM{/ts} <a href="https://github.com/civicrm/civicrm-core/blob/{$version}/release-notes/{$version}.md">{$version}</a>.
```

Comment:
@totten I think it's OK, because the entire div is flipped, so it would become: ".4.7.x civicrm by powered <="